### ISSUE TITLE
[Safari] Audio is muted when playback rate is above 2.0

### ISSUE THREAD

**Issue by eshaang2000** [shaka-project/shaka-player#7289]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes
* did a search for key words: <playbackRate, rate, tempo, trickPlay

**If the problem is related to FairPlay, have you read the tutorial?**

Yes read the tutorial. Issue is on Safari 

**What version of Shaka Player are you using?**

^4.10.2

**Can you reproduce the issue with our latest release version?**
The latest version available is 4.10.13 for my company. Tried with that and was able to reproduce the issue

**Can you reproduce the issue with the latest code from `main`?**
Yes

**Are you using the demo app or your own custom app?**
Custom app

**If custom app, can you reproduce the issue using our demo app?**
No I cannot reproduce on demo app

**What browser and OS are you using?**
Safari - Version 17.6 (19618.3.11.11.5)
Mac - Sonoma - 14.6.1 (23G93

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**

**What are the manifest and license server URIs?**

N/A

**What configuration are you using?  What is the output of `player.getConfiguration()`?**

[Log] Object (app.bundle.js, line 19213)

abr: {enabled: true, useNetworkInformation: true, defaultBandwidthEstimate: 1000000, switchInterval: 8, bandwidthUpgradeTarget: 0.85, …}

abrFactory: function()

ads: {customPlayheadTracker: false, skipPlayDetection: false, supportsMultipleMediaElements: true}

autoShowText: 3

cmcd: {enabled: false, sessionId: "", contentId: "", rtpSafetyFactor: 5, useHeaders: false, …}

cmsd: {enabled: true, applyMaximumSuggestedBitrate: true, estimatedThroughputWeightRatio: 0.5}

drm: {retryParameters: Object, servers: {com.widevine.alpha: "https://mp3localhost.amazon.com:4000/music/player/audible-api/1.0/content/B09K4K5V6L/drmlicense", com.apple.fps.1_0: "https://mp3localhost.amazon.com:4000/music/player/audible-api/1.0/content/B09K4K5V6L/drmlicense"}, clearKeys: {}, advanced: Object, delayLicenseRequestUntilPlayed: false, …}

lcevc: {enabled: false, dynamicPerformanceScaling: true, logLevel: 0, drawLogo: false}

manifest: {retryParameters: Object, availabilityWindowOverride: NaN, disableAudio: false, disableVideo: false, disableText: false, …}

mediaSource: {codecSwitchingStrategy: "smooth", addExtraFeaturesToSourceBuffer: function, forceTransmux: false, insertFakeEncryptionInInit: true, modifyCueCallback: function}

offline: {trackSelectionCallback: function, downloadSizeCallback: function, progressCallback: function, usePersistentLicense: true, numberOfParallelDownloads: 5}

playRangeEnd: Infinity

playRangeStart: 0

preferForcedSubs: false

preferSpatialAudio: false

preferredAudioChannelCount: 2

preferredAudioCodecs: [] (0)

preferredAudioLabel: ""

preferredAudioLanguage: ""

preferredDecodingAttributes: [] (0)

preferredTextLanguage: ""

preferredTextRole: ""

preferredVariantRole: ""

preferredVideoCodecs: [] (0)

preferredVideoHdrLevel: "AUTO"

preferredVideoLabel: ""

preferredVideoLayout: ""

restrictions: {minWidth: 0, maxWidth: Infinity, minHeight: 0, maxHeight: Infinity, minPixels: 0, …}

streaming: {retryParameters: Object, failureCallback: function, rebufferingGoal: 2, bufferingGoal: 10, bufferBehind: 30, …}

textDisplayFactory: function()

textDisplayer: {captionsUpdatePeriod: 0.25}

Object Prototype

**What did you do?**

Basically just said - this.player.trickPlay(2.3)

**What did you expect to happen?**
Playback at 2.3x speed

**What actually happened?**
The stream goes on but no audio is emitted

**Comment by avelad** [comment]
I have tried and I cannot reproduce it.

**Comment by tykus160** [comment]
@eshaang2000 @avelad it seems trickplay does not work as intended in src= mode when playback rate is greater than 2. In MSE mode it is fine.

**Comment by avelad** [comment]
> @eshaang2000 @avelad it seems trickplay does not work as intended in src= mode when playback rate is greater than 2. In MSE mode it is fine.

So it's a limitation of Safari and we can't do anything about it, right?

**Comment by tykus160** [comment]
From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playbackRate):
> The audio is muted when the fast forward or slow motion is outside a useful range (for example, Gecko mutes the sound outside the range 0.25 to 4.0).

**Comment by nkam12** [comment]
> > @eshaang2000 @avelad it seems trickplay does not work as intended in src= mode when playback rate is greater than 2. In MSE mode it is fine.
> 
> So it's a limitation of Safari and we can't do anything about it, right?

From my testing, I believe so. Interestingly, it seems like this is only a limitation for <audio> elements - I was able to get a <video> element working with playbackRate > 2. The native HLS player setting on Shaka on Safari doesn't seem to make any difference either (see https://github.com/shaka-project/shaka-player/issues/3346).
